docs/README.md

```diff
@@ -9,6 +9,7 @@ This repository is a monorepo of the various components required for Chainlink o
 - [Ops](../ops)
 - [Integration/E2E Tests](../tests/e2e)
 - [Demos & Examples](../examples)
+- [Inspecting Terra Contracts](./InspectingContracts.md)
```

Comment:
If you check out the list above notice that this lists different components and folders in the monorepo. Adding a docs page to the same list doesn't really fit. Make a new list of available docs subpages.